python/distributed-ucxx/distributed_ucxx/tests/test_ucxx.py

```diff
@@ -180,7 +180,7 @@ async def test_ucxx_deserialize(ucxx_loop):
     [
         lambda cudf: cudf.Series([1, 2, 3]),
         lambda cudf: cudf.Series([], dtype=object),
-        lambda cudf: cudf.DataFrame([]),
+        lambda cudf: cudf.DataFrame([], dtype=object),
```

Comment:
This one has now arrived with the merge of #125 I think?

Reply:
Yes, I added this in all latest PRs to allow them to complete, it's still showing here because the branch wasn't rebased with latest, I'll do it now and it will go away.